README.md

```diff
@@ -1,29 +1,152 @@
 # MultiModelWizard
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/multi_model_wizard`. To experiment with that code, run `bin/console` for an interactive prompt.
+MultiModelWizard is a way to create and update multiple ActiveRecord models using one form object. Creates a smart object for your wizards or forms. Create one form and form object that can update multiple models with ease.
 
-TODO: Delete this and the text above, and describe your gem
+## Install
 
-## Installation
+Add this to your Gemfile
+```
+  $ gem install multi_model_wizard
+```
 
-Install the gem and add to the application's Gemfile by executing:
+Then run `bundle install` and you're ready to start
 
-    $ bundle add multi_model_wizard
+## Use
```

Comment:
Awesome docs here! I was able to get the project up and running locally very easily :)